java/ql/lib/config/semmlecode.dbscheme

```diff
@@ -442,7 +442,7 @@ isAnonymClass(
 
 #keyset[classid] #keyset[parent]
 isLocalClass(
```

Comment:
Ideally I think we should rename this relation to `isLocalClassOrInterface` or possibly just `isLocalType`. I know it's a bit more work, but leaving it as-is is going to technical debt, as it will be confusing.